docs/clustering.md

```diff
@@ -1,92 +1,168 @@
-## AWX Clustering/HA Overview
+# AWX Clustering Overview
 
-Prior to 3.1, the Ansible Tower HA solution was not a true high-availability system. This system has been entirely rewritten in 3.1 with a focus towards a proper highly-available clustered system. This has been extended further in 3.2 to allow grouping of clustered instances into different pools/queues.
+AWX supports multi-node configurations. Here is an example configuration with two control plane nodes.
 
-* Each instance should be able to act as an entry point for UI and API Access.
-  This should enable AWX administrators to use load balancers in front of as many instances as they wish and maintain good data visibility.
-* Each instance should be able to join the AWX cluster and expand its ability to execute jobs.
-* Provisioning new instance should be as simple as updating the `inventory` file and re-running the setup playbook.
-* Instances can be de-provisioned with a simple management command.
-* Instances can be grouped into one or more Instance Groups to share resources for topical purposes.
-* These instance groups should be assignable to certain resources:
-  * Organizations
-  * Inventories
-  * Job Templates
+```
+       ┌───────────────────────────┐
+       │      Load-balancer        │
+       │   (configured separately) │
+       └───┬───────────────────┬───┘
+           │   round robin API │
+           ▼       requests    ▼
+
+  AWX VM/Pod 1               AWX VM/Pod 2
```

Comment:
@fosterseth This still says "AWX VM".

Reply:
I updated to be more generic `AWX Control Node`